conf/script/ha_setup

```diff
@@ -49,7 +49,7 @@ config() {
     mkdir -p ${HA_CONF}
 
     # TODO Add generation for rule engine to make rule engine schama
-    cp -rf ${SOURCE_HA_CONF}/rules_engine_schema_jbod.json ${HA_CONF}/${HA_CONF_RULE_FILE}
+    rule_engine_config
```

Comment:
Currently we need to keep installing the empty rules engine file for the JBOD in main branch, are we changing that with this change?

Reply:
There was some confusion. I have fixed this now. Thanks !